menpo/feature/visualize.py

```diff
@@ -1,5 +1,7 @@
 import numpy as np
-
+# Deprecated
+#from scipy.misc import imrotate
+from skimage.transform import rotate
```

Comment:
Similarly here - we wouldn't need this import if we removed the glyph view

Reply:
Ok, I think I added all these changes when you said that menpowidgets are no longer supported. Also,  some of the widgets have been implemented in K3D.
```